sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -284,6 +284,21 @@
   /** Controls if the `tracestate` header is attached to envelopes and HTTP client integrations. */
   private boolean traceSampling;
 
+  /** The cache dir. path for caching profiling traces */
+  private @Nullable String profilingTracesDirPath;
+
+  /** Control if profiling is enabled or not for transactions */
+  private boolean profilingEnabled = false;
```

Comment:
Please also add the support for this flag to the Android Manifest, look at ManifestMetadataReader

Reply:
Should I add only profiling enabled as option from manifest with something like io.sentry.traces.profiling-enable?
Don't think the others should be set by the manifest (profilingTracesDirPath, maxTraceFileSize, profilingTracesIntervalMillis), right?

Reply:
Usually, we only add to the AndroidManifest the most important ones, the ones that should be customized the most.
I believe, in this case, only the `profilingEnabled` flag.